Use DRF BooleanFilter as default for DRF FilterSet

When using Django-filter together with Django Rest Framework, shouldn't the `BooleanFilter` for DRF be used as default for a DRF `FilterSet`?

Currently (**Python 2.7.14**, **Django 1.11.6**, **DRF 3.7.0**, **Django-Filter 1.1.0**) I have to manually specify my boolean fields to use the DRF BooleanFilter to accept lowercase `true`/`false`.

I have added some tests here: https://github.com/OskarPersson/django-filter-drf/commit/d9365de19fdf53ce3650e0bf537f8dfd390bd317

Here are the results on my machine. I feel like I'm missing something obvious :/

```
$ ./manage.py test
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
F.F
======================================================================
FAIL: test_filter_lowercase (books.tests.FilterTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/Oskar/git/django_filter_drf/books/tests.py", line 36, in test_filter_lowercase
    self.assertEqual(len(res.data), 1)
AssertionError: 2 != 1

======================================================================
FAIL: test_filterset (books.tests.FilterTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/Oskar/git/django_filter_drf/books/tests.py", line 24, in test_filterset
    self.assertIsInstance(result, filters.BooleanFilter)
AssertionError: <django_filters.filters.BooleanFilter object at 0x10af2fdd0> is not an instance of <class 'django_filters.rest_framework.filters.BooleanFilter'>

----------------------------------------------------------------------
Ran 3 tests in 0.016s

FAILED (failures=2)
```

--- a/django_filters/rest_framework/filterset.py
+++ b/django_filters/rest_framework/filterset.py
@@ -16,6 +16,7 @@ FILTER_FOR_DBFIELD_DEFAULTS = deepcopy(filterset.FILTER_FOR_DBFIELD_DEFAULTS)
 FILTER_FOR_DBFIELD_DEFAULTS.update({
     models.DateTimeField: {'filter_class': IsoDateTimeFilter},
     models.BooleanField: {'filter_class': BooleanFilter},
+    models.NullBooleanField: {'filter_class': BooleanFilter},
 })
 
 

--- a/tests/models.py
+++ b/tests/models.py
@@ -51,6 +51,7 @@ class User(models.Model):
     status = models.IntegerField(choices=STATUS_CHOICES, default=0)
 
     is_active = models.BooleanField(default=False)
+    is_employed = models.NullBooleanField(default=False)
 
     favorite_books = models.ManyToManyField('Book', related_name='lovers')
 

--- a/tests/rest_framework/test_filterset.py
+++ b/tests/rest_framework/test_filterset.py
@@ -31,6 +31,12 @@ class FilterSetFilterForFieldTests(TestCase):
         self.assertIsInstance(result, filters.BooleanFilter)
         self.assertEqual(result.extra['widget'], BooleanWidget)
 
+    def test_booleanfilter_widget_nullbooleanfield(self):
+        field = User._meta.get_field('is_employed')
+        result = FilterSet.filter_for_field(field, 'is_employed')
+        self.assertIsInstance(result, filters.BooleanFilter)
+        self.assertEqual(result.extra['widget'], BooleanWidget)
+
 
 @skipIf(is_crispy(), 'django_crispy_forms must be installed')
 @override_settings(INSTALLED_APPS=settings.INSTALLED_APPS + ('crispy_forms', ))

